lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -905,6 +905,100 @@ static Variable *getUniquelyUsedVariable(Node &node) {
   return dyn_cast<Variable>(&node);
 }
 
+/// Normalize the weight of \p CV with what \p BN is doing.
+/// \return Whether or not the normalization was possible.
+template <typename ElemTy>
+bool normalizeWeights(ConvolutionNode &CV, BatchNormalizationNode &BN) {
+
+  static_assert(
```

Comment:
how is it different from GLOW_ASSERT?

Reply:
That’s a compile time assert. If you try to instantiate this function with a non-supported type, the compiler compiling glow will stop with an error.